city/router.py

```diff
@@ -24,11 +25,13 @@ async def create_city(
             status_code=400, detail="Such name for city already exists"
         )
 
-    return await crud.create_city(db=db, city=city)
+    created_city = await crud.create_city(db=db, city=city)
+    return created_city if created_city else Response(status_code=204)
```

Comment:
you can use status_code parameter in post decorator ;)
Also it should be 201 status code because it's creation